napari/components/overlays/bounding_box.py

```diff
@@ -0,0 +1,32 @@
+from napari.components.overlays.base import SceneOverlay
+from napari.utils.color import ColorValue
+
+
+class BoundingBoxOverlay(SceneOverlay):
+    """
+    Bounding box overlay to indicate layer boundaries.
+
+    Attributes
+    ----------
+    visible : bool
+        If the bounding box is visible or not.
```

Comment:
I do not see where `visible` is defined. Also, I do not see `opacity`, `order`, and `position` in the docstring.

Reply:
These are defined in the base class, and I wasn't sure how to handle it. Should I just copy paste the docstring for those in every overlay?

Reply:
At this moment, unfortunately, you need to copy it. 
It is possible to write a code that will be handling this. But at this moment we do not have it. And it is not so simple.